pkgs/development/libraries/opencv/4.x.nix

```diff
@@ -333,6 +338,21 @@ stdenv.mkDerivation {
   postInstall = ''
     sed -i "s|{exec_prefix}/$out|{exec_prefix}|;s|{prefix}/$out|{prefix}|" \
       "$out/lib/pkgconfig/opencv4.pc"
+  ''
+  # install python distribution information, so other packages can `import opencv`
```

Comment:
I think other packages do `import cv2` and it has always worked with `python3Packages.opencv4`? What hadn't worked is setuptools checking for requirements